pkg/trace/api/api.go

```diff
@@ -532,7 +552,7 @@ func (r *HTTPReceiver) handleGetConfig(w http.ResponseWriter, req *http.Request)
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
-	cfg, err := r.coreClient.ClientGetConfigs(req.Context(), &configsRequest)
+	cfg, err := r.coreClient.ClientGetConfigs(r.coreClientCtx, &configsRequest)
```

Comment:
Setting a context with auth token on the client (GetDDAgentSecureClient) is not enough? 
If not I think it would be cleaner to keep req.Context and add the stored auth token in it instead of sharing the context for the lifetime of the process

Reply:
You're right this is much cleaner.